lms/envs/acceptance.py

```diff
@@ -128,7 +128,6 @@ def seed():
 FEATURES['ENABLE_PAYMENT_FAKE'] = True
 
 # Enable email on the instructor dash
-FEATURES['ENABLE_INSTRUCTOR_EMAIL'] = True
```

Comment:
I feel like this env change is going to break acceptance tests, so heads up for that.